components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -355,9 +354,9 @@ Field Nudging::get_field_out_wrap(const std::string& field_name) {
     const int vec_dim = layout.get_vector_dim();
```

Comment:
fid, layout, vec_dim, and units are all unused vars now, so we should remove them.